docs/nui-tutorials/window.md

```diff
@@ -0,0 +1,388 @@
+---
+layout: default
+title: Nui Tutorial
+---
+[ Home Page ]({{site.baseurl}}/index) <br>
+
+<a name="top"></a>
+# Window tutorial
+
+This tutorial describes how to create and manage windows in NUI, covering the following subjects:
+
+[Overview](#overview)<br>
+[Main Window](#mainwindow)<br>
+[Creating Additional Window](#additionalwindow)<br>
+[Deleting Window](#deleting)<br>
+[Resizing Window](#resizing)<br>
+[Moving Window](#moving)<br>
+[Showing & Hiding Window](#showinghiding)<br>
+[Changing Window Stacking Order](#raisinglowering)<br>
+[Event Handling](#eventhandling)<br>
+[Focus Management](#focusmanagement)<br>
+
+<a name="overview"></a>
+## Window overview
+
+A window contains the visible content of the application.
+
+When an application is created, it automatically creates the [main window](#mainwindow) for building the main user interface of the application.
+
+[Additional windows](#additionalwindow) can be created (as needed) to display additional content.
+
+<img src="{{site.baseurl}}/assets/images/window/MultipleWindow.png">
+
+The window delivers [various events](#eventhandling) (e.g. key event and touch event) to the application.
+
+[Back to top](#top)
+
+<a name="mainwindow"></a>
+## Creating main window
+
+When an application is created, the main window will be automatically created:
+
+~~~{.cs}
+MyApplication myApp = new MyApplication();
+~~~
+
+By default, the main window is created in full screen size, except that if running on the desktop the default size of the main window is set by the **set-env** script. The default main window size can be overriden by environment variables or command line options when launching the application in command line. For example:
```

Comment:
"By default, the main window is created in full screen size, except that if running on the desktop the default size of the main window is set by the **set-env** script."

Thinking this is best mentioned after the  DALI_WINDOW_WIDTH=1920 DALI_WINDOW_HEIGHT=1080 my-app line as don't think most new users will know set-env well.